### PR TITLE
Use ARN from config repository

### PR DESCRIPTION
**Pull request by MarcelMue**
Tested on giraffe.

### CODE REVIEW

**Review by stone-z — APPROVED** [review]
LGTM

**Comment by calvix** [comment]
can we release it as well? so I can  push both changes into 20.0.0?

**Comment by MarcelMue** [comment]
> can we release it as well? so I can push both changes into 20.0.0?

I am waiting for approval from biscuit here:
https://github.com/giantswarm/config/pull/232

**Pull request merged**
